timestamp-impl/src/main/java/com/palantir/timestamp/AvailableTimestamps.java

```diff
@@ -76,7 +76,7 @@ public void refreshBuffer() {
             buffer = currentUpperLimit - lastHandedOut;
             needsRefresh = buffer < MINIMUM_BUFFER || !upperLimit.hasIncreasedWithin(1, TimeUnit.MINUTES);
             if (needsRefresh) {
-                allocateEnoughTimestampsToHandOut(lastHandedOut + ALLOCATION_BUFFER_SIZE);
+                allocateEnoughTimestampsToHandOut(lastHandedOut + ALLOCATION_BUFFER_SIZE, 0L);
```

Comment:
(This is correct, because the point of refreshing the buffer is to ensure that the timestamp _must_ be at least lastHandedOut + buffer.)